apps/api/src/account/account.controller.ts

```diff
@@ -42,4 +43,9 @@ export class AccountController {
   adminRole() {
     return { status: 'OK' }
   }
+
+  @Get('donations')
+  async userDonations(@AuthenticatedUser() user: KeycloakTokenParsed) {
+    return await this.accountService.getDonationsByUser(user.sub as string)
+  }
```

Comment:
Maybe this part is more appropriate to go to `donations.controller.ts` as it's not really part of user profile